app/views/assessment/show.html.erb

```diff
@@ -1,9 +1,11 @@
 <div class="row">
 
+<% if @scores %>
     <div class="col-1"></div>
```

Comment:
this block could potentially be extracted into `app/views/assessment/_scores.html.erb`